web/rest/brand/src/Service/AdministratorRelPublicEntity/SwitchAcl.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Service\AdministratorRelPublicEntity;
+
+use Ivoz\Provider\Infrastructure\Persistence\Doctrine\AdministratorRelPublicEntityDoctrineRepository;
+
+class SwitchAcl
+{
+    public const ACL_GRANT_READ_ONLY = 'grant_read_only';
+    public const ACL_GRANT_ALL = 'grant_all';
+    public const ACL_REVOKE_ALL = 'revoke_all';
+
+    public function __construct(
+        private AdministratorRelPublicEntityDoctrineRepository $administratorRelPublicEntityDoctrineRepository
+    ) {
+    }
+
+    /**
+     * @param int[] $publicEntitiesRelUserIds
+     */
+    public function execute(
+        int $adminId,
+        array $publicEntitiesRelUserIds,
+        string $routeName
```

Comment:
Route is a infraestructure/controller concept. This service shouyd receive the action already resolved